Avoid a boatload of warnings like this:
In file included from ComicalApp.cpp:34:
./../Comical Icons/comical.xpm:3:1: warning: conversion from string literal to 'char *' 
is deprecated [-Wc++11-compat-deprecated-writable-strings]

--- ComicalIcons/comical.xpm.orig	2017-01-06 15:58:55 UTC
+++ ComicalIcons/comical.xpm
@@ -1,5 +1,5 @@
 /* XPM */
-static char *comical_xpm[]={
+static const char *comical_xpm[]={
 "128 128 131 2",
 "Qt c None",
 ".8 c #080000",
